### ISSUE TITLE
Bug at start: Thread 'main' panicked at 'Unexpected rlp error: RlpExpectedToBeData'

### ISSUE THREAD

**Issue by jdetychey** [paritytech/parity#6786]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Parity version**: v1.9.0-unstable-b03117b-20171015/x86_64-linux-gnu/rustc1.21.0
> - **Operating system**: Linux
> - **And installed**:  from source

_ **actual behavior**
> parity can not start
---
Here is the console log:
```
2017-10-16 11:02:25  Starting Parity/v1.9.0-unstable-b03117b-20171015/x86_64-linux-gnu/rustc1.21.0
2017-10-16 11:02:25  Keys path /home/jdetychey/.local/share/io.parity.ethereum/keys/Foundation
2017-10-16 11:02:25  DB path /home/jdetychey/.local/share/io.parity.ethereum/chains/ethereum/db/906a34e69aec8c0d
2017-10-16 11:02:25  Path to dapps /home/jdetychey/.local/share/io.parity.ethereum/dapps
2017-10-16 11:02:25  State DB configuration: fast
2017-10-16 11:02:25  Operating mode: active
2017-10-16 11:02:25  Configured for Foundation using Ethash engine
2017-10-16 11:02:25  Updated conversion rate to Ξ1 = US$340.43 (349697800 wei/gas)

====================

stack backtrace:
   0:     0x563ef689ee1c - backtrace::backtrace::trace::h7d214ae0f4075a7c

Thread 'main' panicked at 'Unexpected rlp error: RlpExpectedToBeData', /checkout/src/libcore/result.rs:906

This is a bug. Please report it at:

    https://github.com/paritytech/parity/issues/new

Abandon (core dumped)
```

**Comment by 5chdn** [comment]
Reset your database :)

Fixed in #6665

Duplicate of #6659 #6749

**Comment by jdetychey** [comment]
thx
> parity db kill
fixed it <3
